src/test/java/com/cloudbees/jenkins/support/SupportTestUtils.java

```diff
@@ -50,7 +50,7 @@ public static String invokeComponentToString(final Component component) {
                     @Override
                     public void add(@CheckForNull Content content) {
                         try {
-                            content.writeTo(baos);
+                            Objects.requireNonNull(content).writeTo(baos);
```

Comment:
You mean to remove the annotation from the methods which are implemented in Tests?

Reply:
Not removing but changing to @NonNull, if we are actually requiring it not to be null.

Reply:
The `Container` is used as base class all over the place. Therefore I suggest to not change the annotation all over the place. My goal was to make the test utils code comply with the annotation. The existing implementation (`@CheckForNull`) says that the caller is allowed to provide `null` and therefore every implementation should check for `null`. If should revert this change let me know, otherwise I suggest to leave it as is.

Reply:
I understand now. Fine then!